web/src/app/alert/AlertContainer.js

```diff
@@ -4,8 +4,8 @@ import SockJS from 'sockjs-client';
 import Stomp from 'stompjs';
 import AlertComponent from './AlertComponent';
 
-const INITIAL_RECALL_DELAY = 500;
-const RECALL_DELAY_INCREMENT = 500;
+const INITIAL_RECALL_DELAY = 500; // how in (milli)  to wait if the first attempt at showing an alert fails
+const RECALL_DELAY_INCREMENT = 500; // time (in ms) added to the recallDelay after each failed attempt
```

Comment:
a variable that needs a comment is a big red flag that a better variable name could be used